network/load-balancer/reference-content/configuring-load-balancer.mdx

```diff
@@ -0,0 +1,137 @@
+
+### Using advanced settings
+
+Here, you have the option to activate **sticky sessions**. When activated, this bind a user's session to a specific server in the pool of backend servers. This ensures that all subsequent sessions from the user are sent to the same backend server while there is at least one active session. Sticky sessions can be **cookie based** or **table based**.
+
+- **Cookie-based** stickiness uses an HTTP cookie to identify a session, and the server in the backend pool that it will stick to. For each incoming request without a stickiness cookie, the Load Balancer creates a cookie and selects a backend for the session. Subsequent requests providing this cookie will land on the same backend. You can customize the name of the cookie.
+- **Table-based** stickiness uses the source (client) IP address to stick sessions to backends.
+
+By default, when activating sticky sessions through the console, the cookie-based method is used when HTTP protocol is selected, and the table-based method is used when TCP protocol is selected. Use the [API](https://developers.scaleway.com/en/products/lb/zoned_api/) if you want to select table-based stickiness with HTTP protocol.
+
+## Configuring a Load Balancer for SSL bridging
+
+SSL bridging allows the user to initiate a secure HTTPS connection with the Load Balancer thanks to the Load Balancer frontend's SSL certificate. The Load Balancer uses its SSL certificate to decrypt incoming HTTPS traffic. This allows the Load Balancer to carry out layer 7 actions on the received traffic. The Load Balancer's backend then initiates a new HTTPS connection to re-encrypt traffic between the Load Balancer and the backend server, this time using the backend server's certificate. 
+
+    <Lightbox src="scaleway-ssl-bridging.webp" alt="" />
+
+To configure your Load Balancer for SSL bridging:
+
+- The frontend should listen on port 443.
+- The frontend must have a [certificate](/network/load-balancer/how-to/add-certificate).
+- The backend should be configured to initiate connections using port 443 and HTTPS protocol.
+- The backend server should have its own [certificate](/network/load-balancer/concepts#certificate).
+
+## Configuring a Load Balancer for SSL offloading
+
+SSL offloading, also known as SSL termination, allows the user to initiate a secure HTTPS connection with the Load Balancer thanks to the Load Balancer frontend's SSL certificate. The Load Balancer uses its SSL certificate to decrypt incoming HTTPS traffic. Layer 7 actions may therefore be applied to the traffic at this stage. Traffic is not re-encrypted on its way from the Load Balancer to the backend server, unlike with SSL bridging. Traffic that has gone through the offloading process is marked with a new header, called `X-Forwarded-Proto`.
+
+    <Lightbox src="scaleway-ssl-offloading.webp" alt="" />
+
+To configure your Load Balancer for SSL offloading:
+
+- The frontend should listen on port 443.
```

Comment:
It is not required to use port 443, you can use whichever port you want, you just need to have a certificate for your frontend.
The frontend must be linked to an HTTP backend (because the fact that the frontend speaks HTTP/HTTPS is linked to the fact that the backend connected to that frontend is either HTTP or HTTPS).

Reply:
corrected